js/apps/admin-ui/src/components/time-selector/TimeSelector.tsx

```diff
@@ -35,17 +35,17 @@ export type TimeSelectorProps = Omit<
     className?: string;
   };
 
-export const getTimeUnit = (value: number | undefined = 0) =>
-  allTimes.reduce(
+const getTimeUnit = (units: TimeUnit[], value: number | undefined = 0) =>
```

Comment:
This can be shortened to the following code by using inferrence:
```suggestion
const getTimeUnit = (units: TimeUnit[], value = 0) =>
```